src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -145,8 +145,22 @@ public async Task RemoveAsync(IAccount account)
 
         private async Task<IEnumerable<IAccount>> GetAccountsInternalAsync(ApiIds apiId, string homeAccountIdFilter = null)
         {
-            var accountsFromCache = await GetAccountsFromCacheAsync(apiId, homeAccountIdFilter).ConfigureAwait(false);
-            var accountsFromBroker = await GetAccountsFromBrokerAsync(homeAccountIdFilter).ConfigureAwait(false);
+            RequestContext requestContext = CreateRequestContext(Guid.NewGuid());
```

Comment:
can this be made from the ServiceBundle? I don't remember exactly.

Reply:
Yes, I think so. But GetAccounts is a bit strange, because it's not a request per se. Anyway, this is just some code copied around.

Reply:
actually, you can remove it, as it's not used in GetAccountsFromBrokerAsync.

---
In reply to: [568137667](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2375#discussion_r568137667) [](ancestors = 568137667)